lnwallet/commitment.go

```diff
@@ -278,6 +278,12 @@ func CommitWeight(chanType channeldb.ChannelType) int64 {
 func HtlcTimeoutFee(chanType channeldb.ChannelType,
 	feePerKw chainfee.SatPerKWeight) btcutil.Amount {
 
+	// For zero-fee HTLC channels, this will always be zero, regardless of
+	// feerate.
+	if chanType.ZeroFeeSecondLevel() {
+		return 0
```

Comment:
EZ 😎

Could also instead have the param be passed in (`0` in this case) if we want to make it a constant so it can be modified easily later. Based on recent discussions though, it seems like the zero fee route may be the best way forward.

Reply:
I prefer zero, since that mitigates the fee leak completely.